src/drivers/Qt/HelpPages.cpp

```diff
@@ -99,15 +99,19 @@ void consoleWin_t::OpenHelpWindow(std::string subpage)
 
 	//printf("Looking for HelpFile '%s'\n", helpFileName.c_str() );
 
-#ifdef WIN32
+#if defined(WIN32)
 	// Windows specific HtmlHelp library function
-	helpWin = HtmlHelp( HWND(winId()), helpFileName.c_str(), HH_DISPLAY_TOPIC, (DWORD)NULL);
+#if defined(_UNICODE) || defined(UNICODE)
+    helpWin = HtmlHelp(HWND(winId()), helpFileName.wstring(), HH_DISPLAY_TOPIC, (DWORD)NULL);
```

Comment:
I don't think std::string has a wstring method either. I'm starting to think that you have not tested this code path at all. I would like the HelpPages.cpp file to be reverted.

Reply:
Something went wrong with the revert of this file. It is showing tons of changes.